tests/integration/tier1/destructive/excluded-packages-removed/main.fmf

```diff
@@ -0,0 +1,15 @@
+summary: |
+    Excluded packages removed
+description: |
+    Verify, that convert2rhel removes packages, which are specified as excluded_pkgs in config.
+    Verification scenarios cover just some of the packages causing the most issues.
+    Those are specified in their respective test plan (remove_excluded_pkgs_epel7 and remove_excluded_pkgs_epel8).
+    Packages are passed to the test as an environment variable through their respective plan.
+    We verify the following packages:
+    epel7: [gnome-documents-libs, shim-x64], epel8: [centos-backgrounds centos-gpg-keys]
```

Comment:
I would probably not mention the packages here as it can cause inconsistency in the future if the packages changes or are added. Also the alma and rocky equivalents are missing